doc/sphinx/addendum/extraction.rst

```diff
@@ -413,6 +420,98 @@ As an example of translation to a non-inductive datatype, let's turn
 
    Extract Inductive nat => int [ "0" "succ" ] "(fun fO fS n -> if n=0 then fO () else fS (n-1))".
 
+Generating FFI Code
+~~~~~~~~~~~~~~~~~~~
+
+The plugin provides mechanisms to generate only OCaml code to
+interface the generated OCaml code with C programs. In order to link compiled
+OCaml code with C code, the linker needs to know
+
+   * which C functions will be called by the ML code (external)
+   * which ML functions shall be accessible by the C code (callbacks)
+
+.. cmd:: Extract Foreign Constant @qualid => @string
+
+   Like :cmd:`Extract Constant`, except that the referenced ML terms
+   will be declared in the form
+
+   ``external @qualid : ML type = "@string"``.
+
+.. caution::
```

Comment:
```suggestion
   .. caution::
```

`.. caution`, `.. exn` and `.. warn` blocks should be indented 3 more spaces to match what we've done elsewhere in the doc.  Looks like there are 7 blocks to adjust.

Reply:
Okay, I will adjust that.